simple_app/utils.py

```diff
@@ -161,16 +175,114 @@ def __init__(self, *args, **kwargs):
         self.db_file: str = kwargs['db_file']
         return
 
+    @staticmethod
+    def multi_param_str_parse(s: str) -> Optional[Tuple[str, str, float]]:
```

Comment:
Not sure that I saw this explicitly tested in test_utils.py; same for `ra_dec_unit_parse`